sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ArithmeticExpressionSuite.scala

```diff
@@ -69,6 +69,7 @@ class ArithmeticExpressionSuite extends SparkFunSuite with ExpressionEvalHelper
 
     checkEvaluation(-c1, -1.1, row)
     checkEvaluation(c1 + c2, 3.1, row)
+    checkDoubleEvaluation(Rand(30), (0.7363714192755834 +- 0.001), row)
```

Comment:
can we create a new test case, instead of adding it to the existing one?

I've been meaning to take the existing one apart for a while.